src/components/OrderCardFulfillmentGroup/OrderCardFulfillmentGroup.js

```diff
@@ -0,0 +1,138 @@
+import React, { Component, Fragment } from "react";
+import PropTypes from "prop-types";
+import Button from "@material-ui/core/Button";
+import Grid from "@material-ui/core/Grid";
+import Typography from "@material-ui/core/Typography";
+import { withStyles } from "@material-ui/core/styles";
+import CartItems from "components/CartItems";
+
+const styles = (theme) => ({
+  fulfillmentGroup: {
+  },
+  fulfillmentGroupDetails: {
+    padding: theme.spacing.unit * 2
+  },
+  fulfillmentGroupCount: theme.typography.subtitle2,
+  fulfillmentGroupHeader: {
+    borderBottom: theme.palette.borders.default,
+    borderTop: theme.palette.borders.default,
+    padding: `${theme.spacing.unit}px ${theme.spacing.unit * 2}px`
+  },
+  fulfillmentGroupHeaderRightColumn: {
+    textAlign: "right"
+  },
+  trackShipmentButton: {
+    textTransform: "none"
+  }
+});
+
+@withStyles(styles)
+class OrderCardFulfillmentGroup extends Component {
+  static propTypes = {
+    classes: PropTypes.object,
+    currentGroupCount: PropTypes.number,
+    fulfillmentGroup: PropTypes.shape({
+      items: PropTypes.shape({
+        nodes: PropTypes.arrayOf(PropTypes.object)
+      }),
+      data: PropTypes.shape({
+        shippingAddress: PropTypes.object
+      })
+    }),
+    hasMoreCartItems: PropTypes.bool,
+    loadMoreCartItems: PropTypes.func,
+    onChangeCartItemsQuantity: PropTypes.func,
+    onRemoveCartItems: PropTypes.func,
+    payments: PropTypes.arrayOf(PropTypes.object),
+    totalGroupsCount: PropTypes.number
+  }
+
+  static defaultProps = {
+    hasMoreCartItems: false,
+    loadMoreCartItems() {},
+    onChangeCartItemsQuantity() {},
+    onRemoveCartItems() {}
+  }
+
+  handleItemQuantityChange = (quantity, cartItemId) => {
+    const { onChangeCartItemsQuantity } = this.props;
+
+    onChangeCartItemsQuantity({ quantity, cartItemId });
+  }
+
+  handleRemoveItem = (_id) => {
+    const { onRemoveCartItems } = this.props;
+
+    onRemoveCartItems(_id);
+  }
+
+  renderItems() {
+    const { classes, fulfillmentGroup } = this.props;
+
+    if (fulfillmentGroup && Array.isArray(fulfillmentGroup.items.nodes)) {
+      return (
+        <Grid className={classes.fulfillmentGroupDetails} item xs={12} md={12}>
+          <CartItems
+            isMiniCart
+            isReadOnly
+            items={fulfillmentGroup.items.nodes}
+          />
+        </Grid>
+      );
+    }
+
+    return null;
+  }
+
+  onTrackShipmentButtonClick() {
+    // TODO: What do we do to track a shipment? Link to a specific provider website?
```

Comment:
Maybe I'll add `trackingUrl` prop on `fulfillmentGroup` in my tracking PR, and then you can render that in a link here? Seems better than recording the carrier and trying to dynamically build links to different carriers.

Reply:
Yeah, that sounds like a good idea.